src/oop/programs_with_multiple_classes/medium/q2/QuestionText.md

```diff
@@ -0,0 +1,39 @@
+# oop-programs_with_multiple_classes-medium-q1
+
+You want to write a program that models furniture within a home. Make a Java program with the following requirements:
+
+
+## Bed
+
+Your `Bed` class should have the following items:
+
+- Attributes:
+    - An `int` called **weightCapacity**
+    - A `boolean` called **isComfy**
+
+- Constructors:
+    - A `public Bed(int, boolean)` constructor to set the 
+  **weightCapacity** and **isComfy** the chair in that order
+
+## Pillow
+
+Your ``Pillow`` class should have the following items:
+
+- Attributes:
+    - A `String` called **filling**
+    - An `int` called **yearMade**
+
+- Constructors:
+    - A `public Pillow(String, int)` constructor to set the 
+    - **filling** and **yearMade** field
+
+## TestRoom
+
+- Your `TestRoom` class will have a `main` method which initializes two pillow and a bed, and prints out the
```

Comment:
We should also maybe be more specific that the values in the sample output are what they need to initialize these objects with.